beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -81,37 +96,53 @@ def lint_job_data(job_data):
 
     if err is None:
         logger.info("Dockerfile Lint check done. Exporting logs.")
-        logs_file_path = export_linter_logs(job_data["logs_dir"], out)
+        # logs file for linter
+        logs_file_path = os.path.join(
+                job_data.get("logs_dir"),
```

Comment:
what about a situation where `logs_dir` key is not set i.e. `None`? Or are we 100% sure of not getting into that scenario?